redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/provision/RedbeamsProvisionEvent.java

```diff
@@ -4,11 +4,15 @@
 import com.sequenceiq.flow.event.EventSelectorUtil;
 import com.sequenceiq.redbeams.flow.redbeams.provision.event.allocate.AllocateDatabaseServerFailed;
 import com.sequenceiq.redbeams.flow.redbeams.provision.event.allocate.AllocateDatabaseServerSuccess;
+import com.sequenceiq.redbeams.flow.redbeams.provision.event.register.RegisterDatabaseServerFailed;
+import com.sequenceiq.redbeams.flow.redbeams.provision.event.register.RegisterDatabaseServerSuccess;
 
 public enum RedbeamsProvisionEvent implements FlowEvent {
     REDBEAMS_PROVISION_EVENT("REDBEAMS_PROVISION_EVENT"),
     ALLOCATE_DATABASE_SERVER_FINISHED_EVENT(EventSelectorUtil.selector(AllocateDatabaseServerSuccess.class)),
     ALLOCATE_DATABASE_SERVER_FAILED_EVENT(EventSelectorUtil.selector(AllocateDatabaseServerFailed.class)),
+    REGISTER_DATABASE_SERVER_FINISHED_EVENT(EventSelectorUtil.selector(RegisterDatabaseServerSuccess.class)),
+    REGISTER_DATABASE_SERVER_FAILED_EVENT(EventSelectorUtil.selector(RegisterDatabaseServerFailed.class)),
```

Comment:
I suspect you're being consistent with other usage elsewhere, but it should either be success/failure or succeeded/failed.

Reply:
Every other service uses finished/failed -- I'd much rather this be consistent with what exists than change it up here.